internal/patroni/config_test.go

```diff
@@ -359,142 +354,28 @@ func TestDynamicConfiguration(t *testing.T) {
 						"another":   "5",
 						"unrelated": "default",
 					},
-					"pg_hba":        []string{},
-					"use_pg_rewind": true,
-					"use_slots":     false,
-				},
-			},
-		},
-		{
-			name: "postgresql.pg_hba: wrong-type is ignored",
-			spec: `{
-				patroni: {
-					dynamicConfiguration: {
-						postgresql: {
-							pg_hba: true,
-						},
-					},
-				},
-			}`,
-			expected: map[string]any{
-				"loop_wait": int32(10),
-				"ttl":       int32(30),
-				"postgresql": map[string]any{
-					"pg_hba":        []string{},
-					"use_pg_rewind": true,
-					"use_slots":     false,
-				},
-			},
-		},
-		{
-			name: "postgresql.pg_hba: default when no input",
-			spec: `{
-				patroni: {
-					dynamicConfiguration: {
-						postgresql: {
-							pg_hba: null,
-						},
-					},
-				},
-			}`,
-			hbas: postgres.HBAs{
-				Default: []*postgres.HostBasedAuthentication{
-					postgres.NewHBA().Local().Method("peer"),
-				},
-			},
-			expected: map[string]any{
-				"loop_wait": int32(10),
-				"ttl":       int32(30),
-				"postgresql": map[string]any{
-					"pg_hba": []string{
-						"local all all peer",
-					},
 					"use_pg_rewind": true,
 					"use_slots":     false,
 				},
 			},
 		},
 		{
-			name: "postgresql.pg_hba: no default when input",
+			name: "HBA pass through",
 			spec: `{
 				patroni: {
 					dynamicConfiguration: {
 						postgresql: {
-							pg_hba: [custom],
+							pg_hba: [calculated, elsewhere],
```

Comment:
These don't come through?

Reply:
This function doesn't do that work anymore. Now,

1. `patroni.PostgresHBAs` reads them out,
2. `postgrescluster.generatePostgresHBAs` stitches them together in the right order, then passes that result
3. here, to `patroni.DynamicConfiguration`.